From 99475da643c32a5409897bad3daa16d0729f8a30 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Mon, 13 May 2019 16:01:57 +0200
Subject: [PATCH] arm: dt: fsl-imx8qxp-tqma8xqp-mbpa8xx: fix ethernet LED

use same binding as out out of tree kernel dt

LED0 / LED3: force output lo
LED1: link / activity, inverted
LED2: GigE link

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts b/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts
index 177ec8915a..9d3aceb4f8 100644
--- a/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts
+++ b/arch/arm/dts/fsl-imx8qxp-tqma8xqp-mbpa8xx.dts
@@ -200,8 +200,10 @@
 			ti,rx-internal-delay = <DP83867_RGMIIDCTL_2_25_NS>;
 			ti,tx-internal-delay = <DP83867_RGMIIDCTL_2_25_NS>;
 			ti,dp83867-rxctrl-strap-quirk;
-			ti,led-function = <0xc100>; /* LED1: Link, LED2: activity */
-			ti,led-ctrl = <0x1001>;     /* active low, LED1/2 driven by phy */
+			/* LED1: Link / activity, LED2: GiGe */
+			ti,led-cfg1 = <0x05b0>;
+			/* 0/3: low, LED1/2 inverted, driven by phy */
+			ti,led-cfg2 = <0x1001>;
 		};
 	};
 };
